delphi-frontend/src/main/java/org/sonar/plugins/communitydelphi/api/ast/AnonymousMethodNode.java

```diff
@@ -41,4 +41,8 @@ public interface AnonymousMethodNode extends ExpressionNode {
   boolean isFunction();
 
   boolean isProcedure();
+
+  CompoundStatementNode getStatementBlock();
+
+  boolean isEmpty();
```

Comment:
What's your reasoning for having `isEmpty()` as a separate function on `AnonymousMethodNode`, when an API consumer could just use the more declarative `getStatementBlock().isEmpty()`?

Reply:
Symmetry with the `RoutineNode` API, mostly.